application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -41,12 +45,24 @@ public TagsCommand(TagSystem tagSystem) {
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
-        // TODO A list might be better than comma separated, which is hard to read
+
+        ArrayList<String> list = new ArrayList<>(tagSystem.getAllIds());
+
+        if (list.size() > 200) {
+
+            Logger logger = Logger.getLogger(TagsCommand.class.getName());
+
+            logger.setLevel(Level.WARNING);
+            logger.warning("- WARNING - TAGS ARE BEYOND 200 LINES ");
+        }
+
         event.replyEmbeds(MessageUtils.generateEmbed("All available tags",
-                String.join(", ", tagSystem.getAllIds()), event.getUser(), TagSystem.AMBIENT_COLOR))
-            .addActionRow(
-                    TagSystem.createDeleteButton(generateComponentId(event.getUser().getId())))
-            .queue();
+                        "* " + String.join("\n ",
+                                list.stream().sorted().collect(Collectors.joining("\n * "))),
```

Comment:
* Dont stuff this into the reply. It is really hard to read. Create the message string before and put it into a variable instead
* Why are you joining multiple times, this looks very confusing and is probably also wrong. The outer join joins only a single string, thats definitely wrong.

Reply:
I found my self in some problem that, 

in 
`String.join("\n ",list.stream().sorted().collect(Collectors.joining("\n * "))` 

it would show the `*` bullet point in all elements except the first one, that is why I had to add `"* "` before the string join

But Il do as you said and remove what is redundant to make it cleaner 👍